src/Scene/AnimationPlayer.js

```diff
@@ -0,0 +1,203 @@
+const FRAMERATE = 60;
+const INTERVAL_TIME = 1000 / FRAMERATE;
+// if is true console.log are enabled to sniff animation'state
+const debugAnimation = false;
+
+// player statut
+const PLAYER_STATE = {
+    // player is stopped
+    STOP: 0,
+    // player plays animation
+    PLAY: 1,
+    // player is at the end of an animation
+    END: 2,
+    // player is paused
+    PAUSE: 3,
+};
+
+const debugMsg =
+    [
+        'Stop',
+        'Play',
+        'End',
+        'Pause',
+    ];
+
+/**
+ * AnimationPlayer
+ * It can play, pause or stop animation
+ */
+function AnimationPlayer(dom) {
```

Comment:
OK thanks for your explanations

> Aren't you instanciating it only from GlobeControls's constructor?

Yes for the moment, but there will be other uses (like earth's rotation, object3D animation...)